docs/conf.py

```diff
@@ -30,6 +30,7 @@
 xarray.CFTimeIndex.__module__ = "xarray"
 
 import xclim  # noqa
```

Comment:
now `xclim` is only used for :
```

# The short X.Y version.
version = xclim.__version__.split("-")[0]
# The full version, including alpha/beta/rc tags.
release = xclim.__version__
```
so perhaps we should also import `__version__` directly?

Reply:
We also access `xclim.indicators` on line 48. I think this is fine as is.